src/components/CweSelector/CweSelector.vue

```diff
@@ -114,15 +104,22 @@ const getUsageClass = (usage: string) => {
           v-for="(cwe, index) in suggestions"
           :key="index"
           class="item gap-1 d-flex justify-content-between"
-          :class="{'selected': index === selectedIndex }"
-          @click.prevent.stop="handleSuggestionClick(abort, `CWE-${cwe.id}`)"
+          :class="{'selected': index === selectedIndex, 'disabled': cwe.usage.toLowerCase() === 'discouraged' }"
+          @click.prevent.stop="cwe.usage.toLowerCase() !== 'discouraged'
```

Comment:
What about abstracting `discouragedCWE` to a function?
Cause we are kind of repeating the same assertion and it would make it sightly easier to read

- Suggestion only for illustrating, it would require the function implementation on setup -
```suggestion
          :class="{'selected': index === selectedIndex, 'disabled': discouragedCwe(cwe) }"
          @click.prevent.stop="!discouragedCwe(cwe)'
```

Reply:
Also i'm seeing now we have ''cwe.isDiscouraged` as property. Couldn't we use that?

Reply:
Actually, this was left behind from a previous iteration.
I replace it with the boolean property `isDiscouraged` no need to write a function to do the check

Good catch! I missed it